doc/1/controllers/index/create/index.md

```diff
@@ -0,0 +1,26 @@
+---
+code: true
+type: page
+title: create
+description: Create an index
+---
+
+# create
+
+Create a new index in Kuzzle
+
+## Signature
```

Comment:
Obsolete template, please use the current one.

For instance: https://docs.kuzzle.io/sdk/go/2/controllers/index/m-delete/